packages/react-async/loom.config.ts

```diff
@@ -1,10 +1,9 @@
-import {createPackage, Runtime} from '@shopify/loom';
+import {createPackage} from '@shopify/loom';
 
 import {quiltPackage} from '../../config/loom';
 
 export default createPackage((pkg) => {
-  pkg.runtimes(Runtime.Browser, Runtime.Node);
-  pkg.entry({root: './src/index'});
-  pkg.entry({name: 'testing', root: './src/testing'});
+  pkg.entry({root: './src/index.ts'});
+  pkg.entry({name: 'testing', root: './src/testing.tsx'});
```

Comment:
Also curious about the use case for the entry file being `testing.tsx` instead of `testing.ts`. Wondering if it's related to using the [`jsdom` testing environment](https://github.com/Shopify/quilt/pull/2150/files#diff-e4cab4a4ad90614b98cde109db889b0ac44ebbfdd533fb09c63a33d55c8e4098R13)?